OpenOversight/app/main/forms.py

```diff
@@ -203,7 +203,7 @@ class AddOfficerForm(Form):
     star_no = StringField('Badge Number', default='', validators=[
         Regexp(r'\w*'), Length(max=50)])
     unique_internal_identifier = StringField('Unique Internal Identifier', default='', validators=[Regexp(r'\w*'), Length(max=50)])
-    job_title = StringField('Job Title')  # Gets rewritten by Javascript
+    job_id = StringField('Job ID')  # Gets rewritten by Javascript
```

Comment:
This change unfortunately breaks the add-officer form. Like instead of a drop-down for the job/rank it just shows a a text field (in which one has to enter the database-id of the job)

Reply:
ooo thanks for catching that @abandoned-prototype! I'll fix that now.